spec/bundler/stub_specification_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+if Bundler.rubygems.provides?(">= 2.1")
+  RSpec.describe Bundler::StubSpecification do
+    let(:with_gem_stub_spec) do
+      stub = Gem::Specification.stubs.first
+      described_class.from_stub(stub)
+    end
+
+    let(:with_bundler_stub_spec) do
```

Comment:
I dont understand the point of this... the `Bundler::StubSpecification` should only ever be passed a `Gem::StubSpecification`